browser.js

```diff
@@ -1,95 +1,105 @@
 module.exports = {
-    plugins: ["@emotion", "react", "jsx-a11y"],
-    env: {
-        browser: true,
-        jest: true,
-    },
-    extends: ["./index.js", "plugin:react/recommended"],
-    rules: {
-        "jsx-a11y/label-has-for": "warn",
-        "jsx-a11y/no-autofocus": "warn",
-        "jsx-a11y/no-noninteractive-element-interactions": "off",
-        "jsx-a11y/no-static-element-interactions": "warn",
-        "jsx-a11y/tabindex-no-positive": "warn",
-        "jsx-a11y/click-events-have-key-events": "warn",
-        "jsx-a11y/anchor-is-valid": "warn",
-        "import/prefer-default-export": "off",
-        "@emotion/jsx-import": "error",
-        "@emotion/no-vanilla": "error",
-        "@emotion/import-from-emotion": "error",
-        "@emotion/styled-import": "error",
-        "react/default-props-match-prop-types": "warn",
-        "react/forbid-prop-type": "off",
-        "react/forbid-prop-types": "warn",
-        "react/jsx-filename-extension": "off",
-        "react/jsx-no-duplicate-props": "warn",
-        "react/no-array-index-key": "warn",
-        "react/no-children-prop": "warn",
-        "react/no-string-refs": "warn",
-        "react/no-unescaped-entities": "warn",
-        "react/no-unused-prop-types": "warn",
-        "react/no-unused-state": "warn",
-        "react/require-default-props": ["warn", { "ignoreFunctionalComponents": true }],
-        "react/self-closing-comp": "warn",
-        "react/sort-comp": 0,
-        "react/style-prop-object": "warn",
-        "react/display-name": "warn",
-        "react/prop-types": "warn",
-        "import/no-extraneous-dependencies": "error",
+  plugins: ['@emotion', 'react', 'jsx-a11y', 'promise'],
+  env: {
+    browser: true,
+    jest: true,
+  },
+  extends: [
+    './index.js',
+    'plugin:react/recommended',
+    'plugin:promise/recommended',
```

Comment:
we tested this against OM or UI?

Reply:
Tested against both.

UI makes no difference.

OM a few trivial things to fix like try catches that do nothing or unused awaits

Reply:
I wanted it to also check for this case:
```
async function () {
await Promise.all([p1, p2])
}
```

If p1 errors before p2 then the Promise.all completes. But because it's inside an async function the catch bubbles up and is meant to be handled by the async function.

I suspect we have a few places where we're doing a Promise.all instead of a Promise.allSettled for triggering async tasks, and if a single tasks fails it can abort the others